ote/src/cljs/ote/views/operator_users.cljs

```diff
@@ -37,8 +37,20 @@
      (tr [:transport-users-page :remove-dialog-text]
        {:user-email (get-in row-info [:member :email]) :operator-name operator-name})]))
 
+(defn- allowed-to-manage?
+  [state]
+  (let [group-id               (get-in state [:params :ckan-group-id])
+        authority-group-id     (get state :authority-group-id)
+        sysadmin?              (get-in state [:user :admin?])
+        transit-authority?     (some-> state :user :groups (get group-id) :transit-authority?)
+        authority-group-admin? (some-> state :user :groups (get authority-group-id) :authority-group-admin?)]
+    (or sysadmin?
```

Comment:
@rckopteri This pairs with that other comment, on frontend this is an `or` instead of `and`, assuming sysadmins can do everything.